package.json

```diff
@@ -60,6 +61,7 @@
     "@babel/preset-typescript": "^7.3.3",
     "@csstools/postcss-global-data": "^1.0.3",
     "@jobber/components": "*",
+    "@jobber/components-native": "*",
```

Comment:
we probably shouldn't install it yet since we don't use em right now.

Reply:
Do we need `esm` for `components-native`? Wouldn't this affect the storybook stuff if it isn't installed?

Reply:
it won't since we don't reference it in the docs. It's esm but still a js file not mjs.

Reply:
as for

> Do we need esm 

If you look at react native's package, they use esm instead of common js. Not sure of RN tree shake things but esm is what's tree shakeable

Reply:
Ahh, good point the storybook stuff is in a separate issue

Also makes sense about the RN package stuff. Thanks for the clarification